smtsolver.ts

```diff
@@ -54,7 +54,10 @@ function solve (query) {
   return solverOutput;
 }
 
-module.exports = {
+export const smtSolver = solve;
+export const availableSolvers = solvers.length;
```

Comment:
What are these for?

Reply:
Only the default export is required, I have now cleaned up this.